src/routes/login/index.ts

```diff
@@ -0,0 +1 @@
+export * from './login.page';
```

Comment:
I understand the desire to (1) do an import on the directory name, while (2) not having your IDE with many tabs all named `index.tsx`.  I did it for a while when I first came to React.  But I find that my IDE names the tabs so they aren't all the same and there isn't any confusion.